### DISCUSSION TITLE
TTS models since Tacotron-2

### DISCUSSION THREAD

**Discussion by swarajdalmia** [coqui-ai/TTS#583] [General Q&A]
Tacotron-2 is one of the most popular models in usage out there. However, the paper was published in 2018. Though it performs well it misses/repeats words occasionally and sometimes produces gibberish as well. There have been lots of models that have come out since then. Which ones would do you think are the best and most stable ones when it comes to a more natural TTS performance and prosody in a conversational setting ? Let's assume we are working with high quality conversational voice recordings.  Or do you think TC2 is still the best one out there.

**Comment by erogol** [comment]
Have you tried any of other alternations of Tacotron2 like Double Decoder Consistency Dynamic Convolution Attention or others? We have many methods for Tacotron implemented in 🐸TTS. 

To my experience, the best model in terms of naturalness with less  tunning is Tacotron. Other models can overdo but they need more tunning.

**Reply by swarajdalmia** [reply]
I have tried Fast-Speech-2 which i felt was worse than TC2. But since TTS models are computationally expensive to train, was looking for suggestions of 2-3 models that have a comparable/better results compared to TC2 to get started with.

**Reply by erogol** [reply]
Then I'd suggest glow TTS or Tacotron2 with DDC

**Comment by astricks** [comment]
I’d have to agree with @erogol, tacotron2 has been by far the easiest to
work with and train correctly. And this is even without DDC.

If your dataset is large enough (20-30 hours to start with) and error-free,
tacotron2 synthesizes accurately once it has learned to attend.

**Reply by swarajdalmia** [reply]
Any other models you'd suggest apart from TC2 ?
